variables.tf

```diff
@@ -1,46 +1,73 @@
 variable "name" {
-  default = "Default"
+  default     = "Default"
+  type        = string
+  description = "Name of the VPC"
 }
 
 variable "project" {
-  default = "Unknown"
+  type        = string
+  description = "Name of project this VPC is meant to house"
 }
 
 variable "environment" {
-  default = "Unknown"
+  type        = string
+  description = "Name of environment this VPC is targeting"
 }
 
 variable "region" {
-  default = "us-east-1"
+  default     = "us-east-1"
+  type        = string
+  description = "Region of the VPC"
 }
 
-variable "key_name" {}
+variable "key_name" {
+  type        = string
+  description = "EC2 Key pair name for the bastion"
+}
 
 variable "cidr_block" {
-  default = "10.0.0.0/16"
+  default     = "10.0.0.0/16"
+  type        = string
+  description = "CIDR block for the VPC"
 }
 
 variable "public_subnet_cidr_blocks" {
-  type    = "list"
-  default = ["10.0.0.0/24", "10.0.2.0/24"]
+  default     = ["10.0.0.0/24", "10.0.2.0/24"]
+  type        = list
+  description = "List of public subnet CIDR blocks"
 }
 
 variable "private_subnet_cidr_blocks" {
-  type    = "list"
-  default = ["10.0.1.0/24", "10.0.3.0/24"]
+  default     = ["10.0.1.0/24", "10.0.3.0/24"]
+  type        = list
+  description = "List of private subnet CIDR blocks"
 }
 
 variable "availability_zones" {
-  type    = "list"
-  default = ["us-east-1a", "us-east-1b"]
+  default     = ["us-east-1a", "us-east-1b"]
+  type        = list
+  description = "List of availability zones"
 }
 
-variable "bastion_ami" {}
+variable "bastion_ami" {
+  type        = string
+  description = "Bastion Amazon Machine Image (AMI) ID"
+}
 
 variable "bastion_ebs_optimized" {
-  default = false
+  default     = false
+  type        = bool
+  description = "If true, the bastion instance will be EBS-optimized"
 }
 
 variable "bastion_instance_type" {
-  default = "t2.micro"
+  default     = "t3.nano"
```

Comment:
This is a backwards incompatible change, but since we're bumping the major version, I figured that now is the time to update it.